lib/minitest.rb

```diff
@@ -836,8 +836,24 @@ def error # :nodoc:
     end
 
     def message # :nodoc:
-      bt = Minitest.filter_backtrace(self.backtrace).join "\n    "
-      "#{self.exception.class}: #{self.exception.message}\n    #{bt}"
+      if self.exception.respond_to?(:cause) && self.exception.cause
+        # Build a chain of exception causes starting from the outermost
+        exc = self.exception
```

Comment:
This has been renamed to `error` with https://github.com/seattlerb/minitest/commit/5802aa08280a3609b684e0ebef95dabfbd3ec9ff